packages/graphql-serve/src/runtime.ts

```diff
@@ -27,12 +29,22 @@ export const createMongoDBClient = async (): Promise<MongoClient> => {
  * Method used to create runtime schema
  * It will be part of the integration tests
  */
-export const createRuntime = async (modelDir: string, db: Db): Promise<GraphbackAPI> => {
+export const createRuntime = async (modelDir: string, db: Db, datasync: boolean): Promise<GraphbackAPI> => {
   const model = await loadModel(modelDir);
 
-  const graphbackAPI = buildGraphbackAPI(model, {
-    dataProviderCreator: createMongoDbProvider(db)
-  })
+  let graphbackAPI;
+  if (datasync) {
+    graphbackAPI = buildGraphbackAPI(model, {
```

Comment:
@ssd71 I think it would be worth to wrap this API into a single helper object as this will rarely will change

Reply:
Agreed! Sounds great!